eng/UpdateTextMate.ps1

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Nit: Extra line at the top.

Reply:
I was giving the comment some breathing room 😛